package.json

```diff
@@ -17,7 +17,7 @@
     "dist"
   ],
   "scripts": {
-    "demo-bundle": "yarn compile:cjs && browserify docs/js/index.js -o docs/demo-bundle.js",
```

Comment:
We still need a way to generate the demo bundle (and now source map too) in order to commit them to the repo under `/docs`.

Reply:
Maybe that's now the equivalent of `compile:cjs`, but I think it makes the contributing instructions easier to follow if there's a clear script to run to generate the bundle, even if it's just now `demo-bundle: "yarn compile:cjs"`.

Reply:
ok